ssl/s3_enc.c

```diff
@@ -558,14 +558,17 @@ int ssl3_enc(SSL *s, int send)
     return (1);
 }
 
-void ssl3_init_finished_mac(SSL *s)
+int ssl3_init_finished_mac(SSL *s)
 {
     if (s->s3->handshake_buffer)
         BIO_free(s->s3->handshake_buffer);
     if (s->s3->handshake_dgst)
         ssl3_free_digest_list(s);
     s->s3->handshake_buffer = BIO_new(BIO_s_mem());
+    if (s->s3->handshake_buffer == NULL)
+        return 0;
     (void)BIO_set_close(s->s3->handshake_buffer, BIO_CLOSE);
+    return (1);
```

Comment:
Yes - might as well while you're at it. There are lots of places where old code doesn't fully conform to the current style guide, so we fix them as we hit them.